hw/ip_templates/clkmgr/dv/env/clkmgr_if.sv.tpl

```diff
@@ -113,6 +118,7 @@ interface clkmgr_if (
   always_comb begin
```

Comment:
I think the `% endif` is meant to go in line 116 since there is no change to jitter enable.